WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -29,6 +29,9 @@ class CompetitionsController < ApplicationController
 
   def new
     @competition = Competition.new
+    if current_user.any_kind_of_delegate?
+      @competition.delegates |= [current_user]
```

Comment:
If I clone a competition that I am already a delegate for and use << then I'll end up with my user id twice in the delegate_ids, but it seems that simple_form handles this so maybe it's not an issue?